lib/presentation/view/order_by/order_by_button.dart

```diff
@@ -60,16 +61,18 @@ class OrderByButtonBuilder {
 
   Widget build() {
     return ListTile(
-      leading: Transform(
-        transform: Matrix4.translationValues(5, 5, 0.0),
-        child: SvgPicture.asset(
-          _currentSorter.orderType.getSortIconPath(imagePath),
-          width: 15,
-          height: 15,
-          fit: BoxFit.fill,
-          color: AppColor.primaryColor,
-        ),
-      ),
+      contentPadding: ResponsiveWidget.isLargeScreen(_context)
+          ? EdgeInsets.symmetric(horizontal: 155)
+          : EdgeInsets.symmetric(horizontal: 16) ,
```

Comment:
srr, I can not understand

Reply:
We need a distance from the left margin on the large tablet screen